types/benches/checksummed_hex_bench.rs

```diff
@@ -0,0 +1,104 @@
+use casper_types::checksummed_hex::{decode, encode, encode_iter};
+use criterion::{black_box, criterion_group, criterion_main, Criterion};
+
+fn encode_and_print_first_5(input: &[u8]) {
+    let encoded = encode(input);
+    format!("{:5}", encoded);
+}
+
+fn encode_iter_and_print_first_5(input: &[u8]) {
+    let encoded = encode_iter(input).take(5).collect::<String>();
+    format!("{:5}", encoded);
```

Comment:
Do we need to do `{:5}` if we already `took` 5?

Reply:
No this is redundant and I'll remove `{:5}`.